### DISCUSSION TITLE
should I pass a UTC+0 DateTimeOffset data to GetNextValidTimeAfter?

### DISCUSSION THREAD

**Discussion by hyzhang** [quartznet/quartznet#2614] [Q&A]
Because in src code, GetNextValidTimeAfter calls GetTimeAfter whose param `DateTimeOffset afterTimeUtc` has a description: "The UTC time to start searching from."https://github.com/quartznet/quartznet/blob/4750150b668371af23700a9640f40046db60ec4c/src/Quartz/CronExpression.cs#L1902
so I think I should pass UTC+0 DateTimeOffset to GetNextValidTimeAfter, just to get some confirmation here. 
And if it's true, it would be better to provide this information in GetNextValidTimeAfter's param description and the document( https://docs.quartz-scheduler.net/apidoc/3.0/html/c22c149c-a2bb-53c1-0a4b-a41ecb71e9f1.htm ).

**Comment by he-dev** [comment]
Were you able to figure this out?
